pyxem/signals/diffraction2d.py

```diff
@@ -1115,6 +1117,50 @@ def template_match_ring(self, r_inner=5, r_outer=7, inplace=False, **kwargs):
             normalize_template_match, template=ring, inplace=inplace, **kwargs
         )
 
+    def filter_signal_axes(self, method="gaussian_filter", **kwargs):
```

Comment:
It needs a docstring.
This is a convenience function which doesn't add any functionality and is a shortcut for

```python
from scipy.ndimage import gaussian_filter
s.map(gaussian_filter, sigma=2)
```

```python
s.filter_signal_axes("gaussian_filter", sigma=2)
```
I find the first one more explicit, would avoid "cluttering" the API with method which doesn't add functionality and also  having different way of doing the same thing.